Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.3
+
+# Jekyll sites often need Node for various things, so we'll install it.
+# Note that Federalist also supports Node, so this just helps recreate
+# same environment used when building the site there.
+
+# https://nodejs.org/en/download/package-manager/
+
+RUN apt-get update && \
+    curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
+    apt-get install -y nodejs zip
```

Comment:
I'm basically doing this in anticipation of using `jekyll_pages_api_search` or some similar strategy for integrating lunr.js with the site.